dev-environment/nginx/templates/default.conf.template

```diff
@@ -0,0 +1,19 @@
+upstream mlflow {
+    server ${MLFLOW_URL};
+}
+
+upstream model {
+    server ${MODEL_URL};
+}
+
+server {
+    listen ${NGINX_PORT};
+
+    location ~* \/invocations$ {
+        proxy_pass      http://model;
```

Comment:
we lost there an information about model name and version. Let's see [here](https://docs.microsoft.com/en-us/azure/databricks/applications/mlflow/model-serving) how the url can look like and try to maybe prepare this configuratoin to redirect for example to 3 different model versions  (every of them have own MODEL_URL_N and some MODEL_VERSION_N and MODEL_NAME_N). If it is not easy to be done then leave it for now with single model